podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -43,9 +62,18 @@ RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain "${RUST_VER
 
 ENV PATH "/root/.cargo/bin:/usr/local/go/bin:$PATH"
 
-RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip && \
-    unzip protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local && rm -f protoc-${PROTOC_VERSION}-linux-x86_64.zip
+RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-${ARCH/s390x/s390_64}.zip && \
+    unzip protoc-${PROTOC_VERSION}-linux-${ARCH/s390x/s390_64}.zip -d /usr/local && rm -f protoc-${PROTOC_VERSION}-linux-${ARCH/s390x/s390_64}.zip
 
 WORKDIR /src
 
+ARG CAA_SRC="https://github.com/confidential-containers/cloud-api-adaptor"
+ARG CAA_SRC_REF="main"
+
+ARG KATA_SRC="https://github.com/kata-containers/kata-containers"
+ARG KATA_SRC_BRANCH="CCv0"
+
+RUN git clone ${CAA_SRC} -b ${CAA_SRC_REF} cloud-api-adaptor
+RUN git clone ${KATA_SRC} -b ${KATA_SRC_BRANCH} kata-containers
+
```

Comment:
With the latest changes, explicit setting of KATA_SRC, CAA_SRC .. is no longer used. I think these should not be added here.
@tumberino @snir911 wdyt ?

Reply:
Agree, make sense

Reply:
@redobed the base structure should be kept same as the other Dockerfiles. And ARG CAA_SRC*, KATA_SRC*, no longer exists in the base Dockerfiles. Please remove it and I think we should be good to merge this.